### PR TITLE
Hugo social media links

### PR DESCRIPTION
**Pull request by sterchelen**
Add github + twitter.

We didn't have specific links to our github and twitter accounts...

### CODE REVIEW

**Comment by pabluk** [comment]
Oh yes! Thanks for catching that :+1:

File: .github/ISSUE_TEMPLATE/new-content.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ about: Submit content to be reviewed and published
 title: Content's title
 labels: content
 assignees: ''
-
+twitter_body: Specific body for twitter
+twitter_hashtags: ['list', 'of', 'hashtags']
```
Comment (pabluk) [inline .github/ISSUE_TEMPLATE/new-content.md:8]: In order to merge this change independently, could you please rebase from current `main` branch without yet the `twitter_*` fields.

File: config.toml [hunk 2]
```diff
@@ -10,6 +10,8 @@ theme = "hyde"
 [Menus]
   main = [
     {Name = "Contributing", URL = "/contribute/"},
+    {Name = "Github", URL = "https://github.com/sre-paris/reliability.re"},
+    {Name = "Twitter", URL = "https://twitter.com/ReliabilityR"}
   ]
```
Comment (pabluk) [inline config.toml:15]: By the way, if you could also include the link to Telegram https://t.me/ReliabilityReport  would be awesome!

File: .github/ISSUE_TEMPLATE/new-content.md (hunk 1 above)
Comment (sterchelen) [reply .github/ISSUE_TEMPLATE/new-content.md:8]: My bad... too early means too much errors :smiley:

**Review by pabluk — APPROVED** [review]

**Comment by pabluk** [comment]
Great! thanks for the enhancement on this @sterchelen

**Pull request merged**